torchvision/csrc/cuda/ROIPool_cuda.cu

```diff
@@ -30,7 +28,7 @@ __global__ void RoIPoolFForward(const int nthreads, const T* bottom_data,
     int roi_end_w = round(offset_bottom_rois[3] * spatial_scale);
     int roi_end_h = round(offset_bottom_rois[4] * spatial_scale);
 
-    // Force malformed ROIs to be 1x1
+    // Force malformed ROIs to be 1x1 or HxW
```

Comment:
by malformed ROIs we meant ROIs with size 0 or smaller, and they become 1x1, not HxW, can you revert this change?

Reply:
Gotcha. Updated.